src/main/java/utils/reports/LoggerReport.java

```diff
@@ -0,0 +1,123 @@
+package utils.reports;
+
+import com.aventstack.extentreports.ExtentTest;
+import com.aventstack.extentreports.Status;
+import com.aventstack.extentreports.model.Media;
+import utils.screenshots.Screenshots;
+
+public class LoggerReport {
+
+    private ExtentTest extentTest;
+
+    public LoggerReport(ExtentTest extentTest) {
+        this.extentTest = extentTest;
+    }
+
+    private Media createMedia(boolean screenShot) {
+        if (screenShot) {
+            return createMedia();
+        }
+        return null;
+    }
```

Comment:
Make this Optional, so you don't return null

Reply:
The setting is made as follows 
   `private Optional<Media> createMedia(boolean screenShot) {
        return Optional.ofNullable( screenShot ? createMedia() : null );
    }
    private Media createMedia() {
        return Report.getReportThreadLocal()
            .createMediaImage(Screenshots.captureScreenShotWithTestStepName());
    }`